src/app/log-in-page/log-in-page.component.html

```diff
@@ -3,8 +3,8 @@
     <form [formGroup]="loginForm" class="login-box">
       <h2>Вход</h2>
       <div class="user-box">
-        <label> Почта</label>
-        <input class="login-box-email" formControlName="email" type="email">
+        <label>Логин</label>
+        <input class="login-box-email" formControlName="name" type="name">
```

Comment:
type="name" looks strange.
need use type="text" here

you can read https://www.w3schools.com/html/html_form_input_types.asp